aws/apigateway/enable-access-logging/metadata.rego

```diff
@@ -0,0 +1,17 @@
+package cloud-metadata.aws.apigateway.enable-access-logging
+
+__rego_metadata__ := {
+	"id": "AWS061",
+	"apiVersion": 2,
+	"version": 1,
+	"title": "API Gateway stages for V1 and V2 should have access logging enabled",
+	"description": "Ensures that Amazon API Gateway API stages have Amazon CloudWatch Logs enabled.",
+	"custom": {
+		"severity": "MEDIUM",
```

Comment:
What governs our use of the various severities? We have a loose guideline [here](https://github.com/aquasecurity/tfsec/blob/master/CONTRIBUTING.md#determining-severity) but it would be good to get Aqua's view on this.